security/advancedtls/advancedtls.go

```diff
@@ -184,13 +184,22 @@ func (o *ClientOptions) config() (*tls.Config, error) {
 		return nil, fmt.Errorf(
 			"client needs to provide custom verification mechanism if choose to skip default verification")
 	}
+	rootCAs := o.RootCACerts
+	if o.VType != SkipVerification && o.RootCACerts == nil && o.GetRootCAs == nil {
```

Comment:
Let's use `o.VType > SkipVerification` to indicate different verification levels. Same for client side.

Reply:
I think if should be `o.VType < SkipVerification` here because enum `VerificationType` is defined in the following way.

```
const (
	CertAndHostVerification VerificationType = iota
	CertVerification
	SkipVerification
)
```

Reply:
Yeah you are correct.